### ISSUE TITLE
Material and Schematics diagrams?

### ISSUE THREAD

**Issue by luisluna2** [vince87/grbl-standalone-lcd#1]
Hello friend, some interesting, have some pictures, material quantity and diagrams of conection?

**Comment by vince87** [comment]
Hi, the pcb project:
https://easyeda.com/editor#id=|8fff9e5f282e4371bb80ce178759623a|2bdadbb8457b4f33bf99c5b31b052e09
the arduino micro sketch:
https://create.arduino.cc/editor/vincenzobini/57ff6378-9ff6-463a-8479-8aac81978e04/preview

**Comment by vince87** [comment]
The progect is an alpha test, not for production!

**Comment by luisluna2** [comment]
@vince87 Yes friend, I will check your project and I will put my best for perfection your work!
